homeassistant/components/apcupsd/sensor.py

```diff
@@ -63,7 +64,7 @@
     "mbattchg": ["Battery Shutdown", PERCENTAGE, "mdi:battery-alert"],
     "minlinev": ["Input Voltage Low", VOLT, "mdi:flash"],
     "mintimel": ["Shutdown Time", "", "mdi:timer-outline"],
-    "model": ["Model", "", "mdi:information-outline"],
+    ATTR_MODEL: ["Model", "", "mdi:information-outline"],
```

Comment:
I don't think it makes sense to use a constant at this place.
It related external information and is not a "Home Assistant" name constant.